### PR TITLE
feature/article list

### LINKED ISSUE #156: Articles list
Add a grid like [mieuxvautartquejamais.com](https://mieuxvautartquejamais.com/)

### PR DESCRIPTION
**Pull request by flexbox**
- rename stuff
- add grid article

Closes #156

### CODE REVIEW

**Comment by MatthysDev** [comment]
@flexbox ready for review

File: src/stylesheets/article.css [hunk 1]
```diff
@@ -27,6 +27,12 @@
   display: inline-block;
   cursor: pointer;
 }
+.img-article {
```
Comment (flexbox) [inline src/stylesheets/article.css:30]:
```
.article-image {
```
is a better name

**Comment by flexbox** [comment]
@MatthysDev Changes requested

### Image preview

- [ ] add `overflow: hidden;` for `.article-image` 


it avoids repaints and do not add extra noise for nothing

image

### SEO for `/articles`

- [ ] add a `h1`
- [ ] replace `h3` with `h2` for article card title

### More article component on `/article/slug`

image

- [ ] It should have the same card style

### homepage `/index` is broken

image

File: src/components/article-list.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      },
+      date: "2021-01-02",
+      image: {
+        url: "https://example.com/image2.jpg",
+      },
+    },
+  },
+]
+
+storiesOf("ArticleList", module).add("default", () => (
```
[the remark this answers is not in the document]
Comment (MatthysDev) [reply src/components/article-list.stories.tsx]: need to fix this before https://github.com/storybookjs/storybook/issues/22796

**Comment by MatthysDev** [comment]
ready for review @flexbox

File: src/components/article-list.stories.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (flexbox) [reply src/components/article-list.stories.tsx:39]: Ok so let's ignore this for now and ship it

**Review by flexbox — COMMENTED** [review]
Sounds Good To Me 👍 

@MatthysDev Ship it I can't approve my own PR

**Pull request merged**